dim_checks/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/dim_checks.yaml

```diff
@@ -12,6 +12,10 @@ dim_config:
     - type: table_row_count
       params:
         condition: ">= 1000"
+    - type: previous_count_avg_within_expected_delta
```

Comment:
@lucia-vargas-a I guess this is closer to what you mentioned in the previous PR.